docs/sdk/dev/troubleshooting.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Troubleshooting
+description: Troubleshooting for the Spryker SDK document.
+template: concept-topic-template
+---
+# Troubleshooting
+
+## `spryker-sdk` command not found
+`spryker-sdk` is an alias that uses the {sdk_folder}/bin/spryker-sdk.sh script.
+If the `spryker-sdk` alias doesn't exist, please add it manually to your shell rc file. Depending on your shell it can be `~/.bashrc`, `~/.zshrc`, `~/.profile`, and other names.
```

Comment:
don't use "please" in tech docs: https://developers.google.com/style/tone#politeness-and-use-of-please